<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>kupagui</class>
 <widget class="QMainWindow" name="kupagui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>kupagui</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>481</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>Command Editor</string>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>461</width>
       <height>161</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="IperfTCP">
      <attribute name="title">
       <string>Iperf-TCP</string>
      </attribute>
      <widget class="QRadioButton" name="tcp_upload">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Upload</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="tcp_download">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Download</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QSpinBox" name="iperf_time">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>40</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
       <property name="value">
        <number>20</number>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>20</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Iperf TIme (s)</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="tcp_mem_server">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>40</y>
         <width>201</width>
         <height>20</height>
        </rect>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string>4096 8192 8388608</string>
       </property>
       <property name="maxLength">
        <number>50</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>20</y>
         <width>201</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>tcp_mem_server</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="tcp_mem_user">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>90</y>
         <width>201</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>4096 8192 8388608</string>
       </property>
       <property name="maxLength">
        <number>50</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>70</y>
         <width>201</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>tcp_mem_user</string>
       </property>
      </widget>
      <widget class="QComboBox" name="tcp_cc">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>90</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string extracomment="Default TCP congestion control">Reno</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BIC</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CUBIC</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Highspeed</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>HTCP</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hybla</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Illinois</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LP</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Scalable</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Vegas</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Veno</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Westwood</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Yeah</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>70</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>tcp_cc</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="IperfUDP">
      <attribute name="title">
       <string>Iperf-UDP</string>
      </attribute>
      <widget class="QRadioButton" name="tcp_download_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Download</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="tcp_upload_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Upload</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="udp_bw">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>60</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
       <property name="singleStep">
        <number>100</number>
       </property>
       <property name="value">
        <number>1000</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>40</y>
         <width>181</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>UDP allocated bandwidth</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="wget">
      <attribute name="title">
       <string>wget</string>
      </attribute>
      <widget class="QLineEdit" name="tcp_mem_user_2">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>90</y>
         <width>201</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>4096 8192 8388608</string>
       </property>
       <property name="maxLength">
        <number>27</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="iperf_time_2">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>40</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
       <property name="value">
        <number>20</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="tcp_mem_server_2">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>40</y>
         <width>201</width>
         <height>20</height>
        </rect>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string>4096 8192 8388608</string>
       </property>
       <property name="maxLength">
        <number>32767</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>70</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>tcp_cc</string>
       </property>
      </widget>
      <widget class="QComboBox" name="tcp_cc_2">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>90</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string extracomment="Default TCP congestion control">Reno</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BIC</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CUBIC</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Highspeed</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>HTCP</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hybla</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Illinois</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Low Priority</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Scalable</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Vegas</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Veno</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Westwood</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Yeah</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>70</y>
         <width>201</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>tcp_mem_user</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>20</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Iperf TIme (s)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>20</y>
         <width>201</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>tcp_mem_server</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="wget_file_size">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>71</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>File Size (Mb)</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QSpinBox" name="user_bw">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>999</number>
     </property>
     <property name="value">
      <number>150</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="server_bw">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>User Bandwidth</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Server Bandwidth</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>210</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Mbps</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>260</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Gbps</string>
     </property>
    </widget>
    <widget class="QComboBox" name="error_model">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>210</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>Rate</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Burst</string>
      </property>
     </item>
    </widget>
    <widget class="QDoubleSpinBox" name="error_rate">
     <property name="geometry">
      <rect>
       <x>281</x>
       <y>210</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="singleStep">
      <double>0.001000000000000</double>
     </property>
     <property name="value">
      <double>0.010000000000000</double>
     </property>
    </widget>
    <widget class="QSpinBox" name="delay">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>210</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
     <property name="value">
      <number>2</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>190</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Error Model</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>190</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Error Rate</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>190</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Delay (ms)</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="button_generate_command">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Command</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="final_command">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>300</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="output_result">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>331</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_run">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>350</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>RUN!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>380</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>save result</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>495</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionLoad_Command"/>
    <addaction name="actionSave_Command"/>
    <addaction name="actionSave_Command_As"/>
    <addaction name="separator"/>
    <addaction name="actionRUN"/>
    <addaction name="actionBye_bye"/>
    <addaction name="actionSave_Result_As"/>
    <addaction name="separator"/>
    <addaction name="actionBye_Bye"/>
   </widget>
   <widget class="QMenu" name="menuXML">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionEdit_XML"/>
    <addaction name="actionEdit_Path"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionGuide"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuXML"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad_Command">
   <property name="text">
    <string>Load Command</string>
   </property>
  </action>
  <action name="actionSave_Command">
   <property name="text">
    <string>Save Command</string>
   </property>
  </action>
  <action name="actionSave_Command_As">
   <property name="text">
    <string>Save Command As..</string>
   </property>
  </action>
  <action name="actionRUN">
   <property name="text">
    <string>Run the command</string>
   </property>
  </action>
  <action name="actionBye_bye">
   <property name="text">
    <string>Save Result</string>
   </property>
  </action>
  <action name="actionSave_Result_As">
   <property name="text">
    <string>Save Result As..</string>
   </property>
  </action>
  <action name="actionBye_Bye">
   <property name="text">
    <string>Bye Bye!</string>
   </property>
  </action>
  <action name="actionEdit_XML">
   <property name="text">
    <string>Edit XML</string>
   </property>
  </action>
  <action name="actionGuide">
   <property name="text">
    <string>Guide</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionEdit_Path">
   <property name="text">
    <string>Edit Path</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
